web/src/user_profile.ts

```diff
@@ -687,12 +788,16 @@ export function show_edit_bot_info_modal(user_id, $container) {
         });
     });
 
-    function edit_bot_post_render() {
+    function edit_bot_post_render(): void {
         $("#edit_bot_modal .dialog_submit_button").prop("disabled", true);
 
-        function get_options() {
+        function get_options(): {
+            name: string;
+            unique_id: number;
+        }[] {
+            assert(bot?.is_bot);
```

Comment:
This assert is a little ambigous in nature. Are we unsure about `bot` not being undefined. 
```
assert(bot!==undefined && bot.is_bot== true);
```
This might be a bit more assertive and readable ??

Reply:
Done!

Reply:
Linter suggests using `assert(bot?.is_bot)` instead.